### ISSUE TITLE
🐛 [BUG] - missing aria-label to prev and next links of is_imageslider

### ISSUE THREAD

**Issue by maofree** [Oksydan/falcon#312]
### Description

Hi
google page speed alert me about an accessiblity problem about a text associated with the links of prev and next of is_imageslider module

normally to fix this type of error I use aria-label="Previous" and aria-label="Next"

![1](https://github.com/Oksydan/falcon/assets/653230/6bfb030f-4247-4d2b-86e1-5e2b72516f94)

### Node.js version

v16

### php version

8.1

### OS and it's version

linux

### Browsers

Chrome

### Required module/theme

is_imageslider

### Reproduction steps

```bash
1. look at the image or try a test with google page speed
```

**Comment by Oksydan** [comment]
Hi @maofree,

thank you for creating an issue. 
It's more like improvements instead of bug. I am going to fix it in next version of is_imageslider module.

**Comment by Oksydan** [comment]
Hi @maofree 

this problem has been fixed here https://github.com/Oksydan/is_imageslider/pull/50. It will be available in next major version.
Thank you for your contribution.
